plugins/generator-1.17.1/forge-1.17.1/triggers/block_break.java.ftl

```diff
@@ -0,0 +1,19 @@
+@Mod.EventBusSubscriber private static class GlobalTrigger {

```

Comment:
I have tried this trigger and it did not work. Did not try other triggers yet, but it seems you did not either.

Reply:
Upon further testing, it seems eg. print works, but placing block on trigger does not, strange

Reply:
Nevermind, I think I know why this happens, I think this part is ok, but the other comment still applies.